libs/components/ag-grid/src/lib/modules/ag-grid/types/lookup-properties.ts

```diff
@@ -9,27 +9,106 @@ import {
+   * @default 0
+   * @deprecated Use the `searchAsync` event emitter and callback instead to provide data to the lookup component.
+   */
   debounceTime?: number;
+  /**
+   * The object property to display in the text input after users select an item in the dropdown list.
+   * @default "name"
+   */
   descriptorProperty?: string;
+  /**
+   * Whether to disable the lookup cell.
+   * @default false
+   */
   disabled?: boolean;
+  /**
+   * Whether to enable users to open a picker where they can view all options.
+   * @default false
+   */
   enableShowMore?: boolean;
+  /**
+   * The object property that represents the object's unique identifier. Specifying this property enables token animations and more efficient rendering. This property is required when using `enableShowMore` and `searchAsync` together.
+   */
   idProperty?: string;
+  /**
+   * Placeholder text to display in the lookup field.
+   */
   placeholderText?: string;
+  /**
+   * The array of object properties to search when utilizing the data property and the built-in search function.
+   * @default ["name"]
+   * @deprecated Use the `searchAsync` event emitter and callback instead to provide data to the lookup component.
+   */
   propertiesToSearch?: string[];
+  /**
+   * The function to dynamically manage the data source when users change the text in the lookup field. The search function must return an array or a promise of an array.
+   * @deprecated Use the `searchAsync` event emitter and callback instead to provide searched data to the lookup component.
+   */
   search?: SkyAutocompleteSearchFunction;
+  /**
+   * Fires when users enter new search information and allows results to be returned via an observable. The event is also fired with empty search text when the "Show more" picker is opened without search text.
+   */
   searchAsync?: (args: SkyAutocompleteSearchAsyncArgs) => void;
+  /**
+   * The array of functions to call against each search result in order to filter the search results when using the data input and the default search function. When using a custom search function via the search property filters must be applied manually inside that function. The function must return `true` or `false` for each result to indicate whether to display it in the dropdown list.
+   * @deprecated  Use the `searchAsync` event emitter and callback instead to provide searched data to the lookup component.
+   */
   searchFilters?: SkyAutocompleteSearchFunctionFilter[];
+  /**
+   * The maximum number of search results to display in the dropdown list. By default, the lookup component displays all matching results. This property has no effect on the results in the "Show more" picker.
+   */
   searchResultsLimit?: number;
+  /**
+   * The template that formats each option in the dropdown list. The lookup component injects values into the template as `item` variables that reference all the object properties of the options.
```

Comment:
```suggestion
   * The template that formats each option in the dropdown list. The lookup component injects values into the template as `item` variables that reference all of the object properties of the search results.
```
In the corresponding searchResultTemplate description above, the description ends with "of the search results. "But here, it ends with "of the options." Should this actually be different? It seems like we mean "of the search results" in both places.